spherical.c

```diff
@@ -111,7 +111,17 @@ int spherical(int index_operation, double yield_strength,
                 printf("Method yet to be implemented\n");
                 break;
             case 3:
-                printf("Method yet to be implemented\n");
+		if(fos_uts*pressure < 0.665*ultimate_strength*weld_efficiency)
+		{
+		t_bpvc= fos_uts*pressure*internal_diameter/\
+		   (2*(2*ultimate_strength*weld_efficiency-0.2*fos_uts*pressure));
+		}
+		else
+		{
+		t_bpvc=(internal_diameter/2)*(exp(0.5*fos_uts*pressure/\
```

Comment:
unable to locate in asme standard. add the section number for better traceability.